src/Nancy.Hal.Tests/JsonResponseProcessorTests.cs

```diff
@@ -11,15 +11,33 @@
 using System.IO;
 using Nancy.Responses;
 using Nancy.Responses.Negotiation;
-using Nancy.Serializers.Json.ServiceStack;
+using Nancy.Serialization.ServiceStack;
+using Nancy.Json;
 using Newtonsoft.Json.Linq;
-using JsonSerializer = Newtonsoft.Json.JsonSerializer;
+using ServiceStack;
 
+using JsonSerializer = Newtonsoft.Json.JsonSerializer;
+using Nancy.Configuration;
 namespace Nancy.Hal.Tests
 {
 
     public abstract class JsonResponseProcessorTests
     {
+        public static INancyEnvironment GetTestingEnvironment()
+        {
+            var environment =
+                new DefaultNancyEnvironment();
+
+            environment.Tracing(
+                enabled: true,
+                displayErrorTraces: true);
+
+            environment.Json();
+            environment.Globalization(new[] { "en-US" });
```

Comment:
Is this required to be set? (Did we introduce a breaking change that users need to be aware of?)

Reply:
It is for the tests to run. Nancy introduced the breaking change in v2 (see https://github.com/NancyFx/Nancy/commit/793104fcc7468cacd90c5bdd4077789fd1f04cd0) that requires a NancyEnvrionment to be added to the context. If you don't set up json, globalization and tracing you will get null ref issues running the tests.